main/WebServer.cpp

```diff
@@ -7893,11 +7893,21 @@ namespace http {
 			}
 
 			char szOrderBy[50];
-			if (order == "")
+			char szQuery[1000];
```

Comment:
I am using strncpy later on to concat this variable. This function takes chars. Is there a equivalent function which takes std::string?